### PR TITLE
Updated Docs for Go Support-Logging and Client

### LINKED ISSUE #803: /docs/support/logging/Ch-Logging.rst is in need of major update
Architectural description (old Java) and how to use logging client are in need of updating.

### PR DESCRIPTION
**Pull request by tsconn23**
Fix #803

Signed-off-by: Trevor Conn <trevor_conn@dell.com>

### CODE REVIEW

File: docs/support/logging/Ch-Logging.rst [hunk 1]
```diff
@@ -46,44 +44,7 @@ High Level Design Architecture
...
-.. image:: EdgeX_SupportingServicesLoggingRequest.png
-
-**Sequence Diagram for Query Historical Logging**
-
-.. image:: EdgeX_SupportingServicesQueryHistoricalLogging.png
-
-**Sequence Diagram for Removing Historical Logging**
-
-.. image:: EdgeX_SupportingServicesRemoveHistoricalLogging.png
+The above diagram shows the high-level architecture for EdgeX Foundry Logging Service. Other microservices interact with EdgeX Foundry Logging Service through RESTful APIs to submit their logging requests, query historical logging, and remove historical logging. Internally, EdgeX Foundry's Logging Service utilizes the .. _GoKit logger: https://github.com/go-kit/kit/tree/master/log as its internal logging framework. Two configurable persistence options exist supported by EdgeX Foundry Logging Service: file or MongoDB.
```
Comment (mhall119) [inline docs/support/logging/Ch-Logging.rst]: Hyperlinks need to be defined as:

\`Link Text \<URL\>\`_

So this should be:

\`GoKit logger \<https://github.com/go-kit/kit/tree/master/log>`_

File: docs/support/logging/Ch-Logging.rst [hunk 2]
```diff
@@ -147,62 +112,75 @@ Configuration Properties
...
 
 ====================================================
-Logging Service Client Library for Java (Deprecated)
+Logging Service Client Library for Go
 ====================================================
 
-As most of EdgeX Foundry microservices are implemented in Java, we provide a Client Library for Java, so that Java-based microservices could directly switch their Loggers to use EdgeX Foundry Logging Service.  The next graphic shows the high-level design architecture for the Java Client Library.
+As the reference implementation of EdgeX Foundry microservices is written in Go, we provide a Client Library for Go so that Go-based microservices could directly switch their Loggers to use the EdgeX Foundry Logging Service.
+
+The Go LoggingClient is part of the .. _go-mod-core-contracts module: https://github.com/edgexfoundry/go-mod-core-contracts. This module can be imported into your project by including a reference in your go.mod. You can either do this manually or by executing "go get github.com/edgexfoundry/go-mod-core-contracts" from your project directory will add a reference to the latest tagged version of the module.
```
Comment (mhall119) [inline docs/support/logging/Ch-Logging.rst]: Link format needs to be updated as described in comment above

File: docs/support/logging/Ch-Logging.rst (hunk 1 above)
Comment (tsconn23) [reply docs/support/logging/Ch-Logging.rst]: Change was made

File: docs/support/logging/Ch-Logging.rst (hunk 2 above)
Comment (tsconn23) [reply docs/support/logging/Ch-Logging.rst]: Change was made

**Pull request merged**